input/blog/2021-08-29-cake-v1.2.0-released.md

```diff
@@ -0,0 +1,93 @@
+﻿---
+title: Cake v1.2.0 released
+category: Release Notes
+author: devlead
+---
```

Comment:
Add `releaseName` metadata, so that the blog post is linked on home page (see https://github.com/cake-build/website/blob/master/input/blog/2021-03-06-cake-v1.1.0-released.md for an example)

Reply:
Ah then fetch contributors should be updated with this too.

I'll take a look